controllers/uninstall.go

```diff
@@ -33,6 +33,10 @@ var (
 				types.NamespacedName{Name: "kui-web-terminal-sub", Namespace: m.Namespace},
 				schema.GroupVersionKind{Group: "apps.open-cluster-management.io", Kind: "Subscription", Version: "v1"},
 			),
+			newUnstructured(
```

Comment:
I believe this uninstallation occurs once the rest of the upgrade is complete. Does waiting to delete the appsub cause conflicts when the templates are applied? I think it would be better to delete the appsub first then redeploy the chart.

Reply:
I see now how that would be the case. Instead, I suppose I should make a new list of appsubs to be deleted towards the top of the reconcile function, to be added to as more components are moved to the rendering function?

Reply:
I imagine there is already a list for early cleanup for those appsubs which we migrated to MCE